SalesforceSDK/Salesforce.SDK.Core/Source/Auth/AuthStorageHelper.cs

```diff
@@ -420,5 +421,94 @@ internal void DeleteData(string key)
                 _persistedData.Values.Remove(key);
             }
         }
+
+        internal void PersistEncryptionSettings(string password, string salt, string nonce = null)
```

Comment:
Why do you need the 3rd param here (string nonce)? you don't seem to be using it anywhere inside the method!